mea/Cargo.toml

```diff
@@ -13,32 +13,47 @@
 # limitations under the License.
 
 [package]
+authors = ["tison <wander4096@gmail.com>"]
```

Comment:
```suggestion
```

Somehow this is deprecated metadata and it contributes to trouble more than help :D